rebasehelper/application.py

```diff
@@ -351,7 +351,7 @@ def patch_sources(self, sources):
         git_helper = GitHelper(sources[0])
         if not self.conf.non_interactive:
             git_helper.check_git_config()
-        patch = Patcher(self.conf.patchtool)
+        patch = Patcher("git")
```

Comment:
I think the Patcher class should be completely removed and GitPatchTool used directly. To minimize the need to refactor the code you could rename the ```run_patch()``` to ```patch()```. The Patcher tool was mostly a wrapper to allow us to execute different patch tools in the same manner.